### DISCUSSION TITLE
Remacs Vertico improvement

### DISCUSSION THREAD

**Discussion by zjw12942151** [browneyedsoul/remnote-plugins#65] [Ideas]
I quite like this plugin personally, however, i have a suggestion to improve it functionality.
The search plugin now is bigger which is more easier for me to search rem and read through the whole rem instead of a glimpse of that, but when i am going to hyperlink some word, the hyperlink button do not show the Remacs Vertico. 
So i suggest you can edit it and let the hyperlink function to be working with the Remacs Vertico as well!
Thank you! Is it possible?

**Comment by browneyedsoul** [comment]
Hey! Thanks for your kind feedback! I'm just curious — what’s the hyperlink function you mentioned? Could you elaborate a bit more with some images? :)

**Reply by zjw12942151** [reply]
In remnote it might be called ‘reference’. It’s just to create a reference which you can refer to other rem by hovering it!

**Reply by browneyedsoul** [reply]
It seems like you might want to see the reference cadidate in the popup also looks like vertico result as well when user type the `[[` right?

**Reply by zjw12942151** [reply]
yes you are correct, sorry that i forgot to attach the images here, but that's what i mean! because the pop-up windows is too small to prevent the bettter visuality, and also for the hyperlink in the image below:
<img width="412" alt="Screenshot 2024-11-02 at 4 42 48 PM" src="https://github.com/user-attachments/assets/41dffde0-895b-4a20-8b52-9fa22c949999">

**Reply by zjw12942151** [reply]
<img width="405" alt="Screenshot 2024-11-02 at 4 44 35 PM" src="https://github.com/user-attachments/assets/365de96b-4d9f-4aea-974f-a069adc7d225">
which is thie link function

**Reply by browneyedsoul** [reply]
I’ll take a look at this feature, but please don’t expect too much in the near future. I’m just evaluating how easily this feature can be applied without affecting other core functions in RemNote. Anyway, thanks for your feedback!

**Reply by zjw12942151** [reply]
thanks Park!
